### PR TITLE
PS-7270: extra_port should not accept proxied connections - 5.7

### PR DESCRIPTION
**Pull request by imp3142**

### CODE REVIEW

**Comment by altmannmarcelo** [comment]
https://ps57.cd.percona.com/job/percona-server-5.7-param/334/

File: mysql-test/t/proxy_protocol_extra_port.test [hunk 1]
```diff
@@ -0,0 +1,13 @@
+--source include/not_embedded.inc
+SELECT @@proxy_protocol_networks;
```
Comment (percona-ysorokin) [inline mysql-test/t/proxy_protocol_extra_port.test]: Change this to `assert.inc`

File: mysql-test/t/proxy_protocol_extra_port.test [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+--source include/count_sessions.inc
+
+--echo # An unproxied client connection should be accepted when connection to an extra_port if proxying enabled on the server
+--disable_query_log
```
Comment (percona-ysorokin) [inline mysql-test/t/proxy_protocol_extra_port.test]: Either remove this completely or add corresponding `--enable_query_log`

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

**Comment by altmannmarcelo** [comment]
@percona-ysorokin  https://ps57.cd.percona.com/job/percona-server-5.7-param/335/ 
`proxy_protocol_admin_port` test is passing on all platforms.

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**